README.md

```diff
@@ -51,9 +51,9 @@ Where `llvm_build_dir` is the LLVM build directory.
 
 The translator can be built as a regular LLVM subproject. To do that you need to clone it to `llvm/projects` or `llvm/tools` directory. 
 ```
-git clone http://llvm.org/git/llvm.git
+git clone http://llvm.org/git/llvm.git release_90
```

Comment:
In `.travis.ci.yml` this branch is named differently (`release/9.x`). Why?

Reply:
We switched to llvm monorepo for travis builds.  But documentation seems to be outdated.